_usage/mvu.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Performing a major version upgrade
+nav_order: 9
+---
+
+## Performing a major version upgrade
+
+Babelfish supports a major version upgrade (MVU) from implementations deployed with a Babelfish PostgreSQL server version 13.6 or later. Currently, the Babelfish PostgreSQL server only supports an upgrade path using the `pg_upgrade` module. During the upgrade, you will need to:
+
+- Perform a major version upgrade of the PostgreSQL server.
+- Update the Babelfish extensions to the target version.
+- Confirm that the Babelfish-specific GUCs are the same in the source and target versions.
+
+Before beginning an upgrade, we strongly recommend that you create a copy of the old PostgreSQL `data` directory in case you have trouble and need to revert to it.
+
+Babelfish supports a major version upgrade on Babelfish PostgreSQL servers version 13.6 or later. You can use the following releases as an upgrade source (the server from which you'll upgrade):
+
+- Babelfish 1.2.2, built with PostgreSQL 13.6
+- Babelfish 1.3.1, built with PostgreSQL 13.7
+- Babelfish 1.4.0, built with PostgreSQL 13.8
+- Babelfish 1.5.0, built with PostgreSQL 13.9
+
+The target server must be running Babelfish PostgreSQL servers, version 14.5 or later. You can use the following release as an upgrade target (the version to which you'll upgrade):
```

Comment:
This line might be confusing for the users as they might assume that target server should have Babelfish installed, which is not correct. I think we should just say that:
```
You can use the following release as an upgrade target (the version to which you'll upgrade):
```

Reply:
it should be 14.6 or later

Reply:
Now reads:

You can use the following Babelfish release as an upgrade target (the version to which you'll upgrade):

- Babelfish 2.3.0, built with PostgreSQL 14.6